generate.sh

```diff
@@ -161,6 +161,16 @@ if [ $? -ne 0 ]; then
 fi
 print_log "Documentation Generated Successfully!\n"
 
+if [ -n "$APIDOCS_NAME" ] && [ -n "$APIDOCS_URL" ]; then
+  if [ "$APIDOCS_NAME" == "swagger" ]; then
+    cd ..
+    git clone -q https://github.com/swagger-api/swagger-ui.git swagger >/dev/null 2>&1
+    cd yaydoctemp/_build/html
+    cp -r ${BUILD_DIR}/../swagger/dist apidocs
```

Comment:
Instead of this clone directly and just move the directory to _build/html. The advantage is that you won't have to use the name `yaydoctemp` as BUILD_DIR is an abstraction for that.